Source/Renderer/TextureMagnificationFilter.js

```diff
@@ -7,12 +7,38 @@ define([
     'use strict';
 
     /**
-     * @private
+     * Enumerates all possible filters used when magnifying WebGL textures, which takes places when zooming
+     * into imagery. Provides the possible values for the {@link ImageryLayer#magnificationFilter} property.
+     *
+     * @exports TextureMagnificationFilter
+     *
+     * @see TextureMinificationFilter
+     * @see ImageryLayer#magnificationFilter
      */
     var TextureMagnificationFilter = {
+        /**
+         * Nearest neighbor sampling of image pixels to texture.
```

Comment:
In this file and below, try to avoid providing reference doc that is just a repeat of the enum.  @lilleyse could you add a tad more description and trade offs, e.g., visual quality vs speed?